scripts/test/e2e/wrapper

```diff
@@ -2,10 +2,11 @@
 # Setup, run and teardown e2e test suite in containers.
 set -eu -o pipefail
 
+echo "exp $DOCKERD_EXPERIMENTAL"
```

Comment:
Is this leftover debug? If not then due to `set -u` above if `DOCKERD_EXPERIMENTAL` is not set it'll error out I think.